cpu/ezr32wg/Makefile

```diff
@@ -4,4 +4,7 @@ MODULE = cpu
 # add a list of subdirectories, that should also be build
 DIRS = periph $(RIOTCPU)/cortexm_common
 
+#
```

Comment:
it looks like something is missing without a text.
What happens when you compile vectors.c with LTO? Do you know if it is enough to add a attribute used tags somewhere or if there's more to it?